variables.tf

```
@@ -21,10 +21,16 @@ variable "allowed_aws_services_for_sns_published" {
 
 variable "kms_master_key_id" {
   type        = string
-  description = "The ID of an AWS-managed customer master key (CMK) for Amazon SNS or a custom CMK"
+  description = "The ID of an AWS-managed customer master key (CMK) for Amazon SNS or a custom CMK."
   default     = "alias/aws/sns"
 }
 
+variable "use_encryption" {
```

```suggestion
variable "encryption_enabled" {
```

We use the pattern of naming flags `<feature>_enabled`

---

updated